lua/compe/config.lua

```diff
@@ -18,6 +18,10 @@ Config.setup = function(config)
   config.source_timeout = config.source_timeout or SOURCE_TIMEOUT
   config.incomplete_delay = config.incomplete_delay or INCOMPLETE_DELAY
   config.allow_prefix_unmatch = Config._true(config.allow_prefix_unmatch)
+  config.autocomplete = Config._true(config.autocomplete)
```

Comment:
`_true` will normalize `nil` as `false`.
So I think the below condition always returns false, right?

Reply:
yeah you're right..

pushed a fix

Reply:
or maybe `_false` function that defaults to true on anything other than `true` or `0` would be cleaner?